docs/gallery/domain/brain_observatory.py

```diff
@@ -71,8 +73,9 @@
         data=dataset.get_stimulus_template(stimulus),
         unit="NA",
         format="raw",
-        timestamps=[0.0],
+        rate=np.nan,
```

Comment:
Timestamps does not make sense here because the images are repeated:

![image](https://github.com/user-attachments/assets/1c24bdf9-f34a-4fea-b974-0593e68a35b1)

From the data, the rate is not very clear, I think it could be derived from the fluorescence timestamps but I don't really have the time to explore the dataset to that level.